<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 75, 82);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 201, 126);
font:42pt &quot;Microsoft JhengHei UI Light&quot;;
</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="familyName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);


background:transparent;
border:none;
color:rgb(255, 255, 255);;
border-bottom:1px solid rgb(96, 118, 129);</string>
   </property>
   <property name="placeholderText">
    <string>Family name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);


background:transparent;
border:none;
color:rgb(255, 255, 255);;
border-bottom:1px solid rgb(96, 118, 129);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>First name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="creatAcc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(0, 128, 128);
font: 25 22pt &quot;Microsoft YaHei UI Light&quot;;
color: rgb(222, 184, 135);
border-style: dotted;}
QPushButton:hover{
background-color:rgb(241, 241, 241);
font: 25 22pt &quot;Microsoft YaHei UI Light&quot;;
color: rgb(0, 159, 159);
border-style: dotted;}
</string>
   </property>
   <property name="text">
    <string>Sing up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pass_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);


background:transparent;
border:none;
color:rgb(255, 255, 255);;
border-bottom:1px solid rgb(96, 118, 129);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);


background:transparent;
border:none;
color:rgb(255, 255, 255);;
border-bottom:1px solid rgb(96, 118, 129);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Email Adress</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Pass_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);


background:transparent;
border:none;
color:rgb(255, 255, 255);;
border-bottom:1px solid rgb(96, 118, 129);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Repeat Password</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Male">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:11pt &quot;Microsoft JhengHei UI Light&quot;;
color:rgb(255, 201, 126);
</string>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Female">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:11pt &quot;Microsoft JhengHei UI Light&quot;;
color:rgb(255, 201, 126);</string>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backtologin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(0, 128, 128);
font: 25 22pt &quot;Microsoft YaHei UI Light&quot;;
color: rgb(222, 184, 135);
border-style: dotted;}
QPushButton:hover{background-color:rgb(241, 241, 241);
font: 25 22pt &quot;Microsoft YaHei UI Light&quot;;
color: rgb(222, 184, 135);
border-style: dotted;}</string>
   </property>
   <property name="text">
    <string>‚ùÆ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invalidLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(238, 74, 38);
border-style: dotted;
}
QPushButton:hover{font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(241, 241, 241);
border-style: dotted;}
</string>
   </property>
   <property name="text">
    <string>INVALID PASSWORD!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invalidLabel_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(238, 74, 38);
border-style: dotted;
}
QPushButton:hover{font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(241, 241, 241);
border-style: dotted;}
</string>
   </property>
   <property name="text">
    <string>INVALID PASSWORD!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invalidfirstname">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(238, 74, 38);
border-style: dotted;
}
QPushButton:hover{font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(241, 241, 241);
border-style: dotted;}
</string>
   </property>
   <property name="text">
    <string>First Name Must Be 3-15 Charchter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invalidfamilyname">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(238, 74, 38);
border-style: dotted;
}
QPushButton:hover{font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(241, 241, 241);
border-style: dotted;}
</string>
   </property>
   <property name="text">
    <string>First Name Must Be 3-15 Charchter</string>
   </property>
  </widget>
  <zorder>invalidfamilyname</zorder>
  <zorder>invalidfirstname</zorder>
  <zorder>invalidLabel</zorder>
  <zorder>invalidLabel_2</zorder>
  <zorder>label</zorder>
  <zorder>familyName</zorder>
  <zorder>firstName</zorder>
  <zorder>creatAcc</zorder>
  <zorder>Pass_2</zorder>
  <zorder>Email</zorder>
  <zorder>Pass_3</zorder>
  <zorder>Male</zorder>
  <zorder>Female</zorder>
  <zorder>backtologin</zorder>
 </widget>
 <tabstops>
  <tabstop>firstName</tabstop>
  <tabstop>familyName</tabstop>
  <tabstop>Email</tabstop>
  <tabstop>Pass_2</tabstop>
  <tabstop>Pass_3</tabstop>
  <tabstop>creatAcc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
